cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -31,7 +44,12 @@ def test_vmware_provider_filters():
             2.All hosts are listed.
             3.We should have at least 3 filters based on VMware version.
     """
-    pass
+    esx_platforms = ['Platform / ESX 6.0', 'Platform / ESX 6.5', 'Platform / ESX 6.7']
+    view = navigate_to(appliance.collections.hosts, 'All')
+    all_options = view.filters.navigation.all_options
+    logger.info("All options for Filters are: {} ".format(all_options))
+    for esx_platform in esx_platforms:
```

Comment:
**Optional** Since esx_platforms is only used once, thoughts on:
```
for esx_platform in ['Platform / ESX 6.0', 'Platform / ESX 6.5', 'Platform / ESX 6.7']:
    soft_assert(esx_platform in all_options, "ESX Platform does not exists in options")
```

Reply:
It will work but I personally liked idea (habit from C) of defining it upfront so in case I need to change entries there, I can do it without touching it anywhere else in logic below, even if its only one place. Either is ok for me.